rialto_airflow/dags/harvest.py

```diff
@@ -60,17 +61,19 @@ def dimensions_harvest_dois(snapshot):
         """
         Fetch the data by ORCID from Dimensions.
         """
+        authors_csv = snapshot.path / "authors.csv"
         pickle_file = snapshot.path / "dimensions-doi-orcid.pickle"
-        dimensions.doi_orcids_pickle(pickle_file, limit=dev_limit)
+        dimensions.doi_orcids_pickle(authors_csv, pickle_file, limit=dev_limit)
```

Comment:
these `authors_csv` changes are one of the parts of this i'm least sure about, but afaict, this is a legit signature mismatch that the type checker caught, and it doesn't seem to be covered by tests?  it seemed to me that the path was what was wanted and not the csv content, but i walso wasn't 100% sure about that.

Reply:
I think the risk here is minimal since I don't think we are not going to be keeping this code.